recipes-ni/ni-safemode-utils/ni-safemode-utils_1.0.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "nilrt safemode utilities"
+DESCRIPTION = "nilrt distro-specific safemode utilities that provide basic system functionality."
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SECTION = "base"
+
+SRC_URI = "\
+	file://nisafemodeversion \
+	file://niinstallsafemode \
+	file://nicompareversion \
+"
+
+natinstbin="/usr/local/natinst/bin"
+
+FILES_${PN} += "\
+	${natinstbin}/nisafemodeversion \
+	${natinstbin}/niinstallsafemode \
+	${natinstbin}/nicompareversion \
+"
+
+DEPENDS += "shadow-native pseudo-native niacctbase update-rc.d-native"
+
+RDEPENDS_${PN} += "niacctbase bash"
+
+RDEPENDS_${PN}_append_x64 += "fw-printenv"
```

Comment:
Is it reasonable to not have the _append_x64 since these recipes are now exclusively for x64 systems?

Reply:
Sure, I'll remove it